detectable/src/main/java/com/synopsys/integration/detectable/detectables/dart/pubdep/PubDepsParser.java

```diff
@@ -15,6 +13,17 @@
 public class PubDepsParser {
     private static final String UNRESOLVED_VERSION_SUFFIX = "...";
 
+    // starting with Dart 2.19 the dependency graph contains UTF-8 characters instead of ASCII-only
+    // the following are some of the characters that are used and are required for parsing of the graph
```

Comment:
Are there other characters that we should also handle later?

Reply:
The following characters are used in the tree:

```
├     E2 94 9C       U251C
└     E2 94 94       U2514
─     E2 94 80       U2500
│        E2 94 82       U2502
```

Having said that, with the way the tree is generated now it's not a requirement to take all of them into account.